pkg/github/actions/client.go

```diff
@@ -97,7 +97,7 @@ func (client *ActionsClient) updateTokenIfNeeded(ctx context.Context) error {
 		return fmt.Errorf("failed to get app access token on refresh: %w", err)
 	}
 
-	authInfo, err := auth.GetAuthorizationInfo(ctx, accessToken, client.gitHubConfig)
+	authInfo, err := auth.GetAuthorizationInfo(ctx, accessToken, &client.envData.GitHubAPIUrl, client.gitHubConfig)
```

Comment:
Why do we have to pass a pointer here? I believe just passing the value would be enough